lib/inferno/dsl/fhir_evaluator/dataset.rb

```diff
@@ -0,0 +1,34 @@
+module FhirEvaluator
+  # A Dataset is an Array of FHIR data to be summarized or evaluated,
+  # with convenience methods for loading from a file path or from an
+  # Array of FHIR JSON strings.
+  class Dataset < Array
```

Comment:
I don't see a reason for this to subclass Array. Make it something like `DataSetLoader` instead which just returns a plain Array.

Reply:
Addressed